tests/slurm-tests/slurm_test_llama_nemotron_super_49B_v1._5_evals/check_eval_results.py

```diff
@@ -0,0 +1,258 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import io
+import json
+import os
+
+REASONING_TASKS = [
+    "math-500",
+    "aime24",
+    "aime25",
+    "gpqa",
+    "mmlu-pro",
+    "livecodebench",
+    "scicode",
+    "hle",
+]
+
+REASONING_BENCHMARKS_SCIENCE_HLE = {"scicode", "hle"}
+
+REASONING_REQUIRED_FIELDS = {
+    "math-500": ["symbolic_correct"],
+    "aime24": ["symbolic_correct"],
+    "aime25": ["symbolic_correct"],
+    "gpqa": ["symbolic_correct"],
+    "mmlu-pro": ["symbolic_correct"],
+    "livecodebench": ["accuracy"],
+    "scicode": ["problem_accuracy", "subtask_accuracy"],
+    "hle": ["judge_correct", "symbolic_correct"],
+}
+
+REASONING_METRIC_RANGES = {
+    "reasoning_on": {
+        "math-500": (90.0, 100.0),
+        "aime24": (84.0, 94.0),
+        "aime25": (78.0, 88.0),
+        "gpqa": (70.0, 80.0),
+        "mmlu-pro": (75.0, 85.0),
+        "livecodebench": (65.0, 75.0),
+        "scicode": {
+            "problem_accuracy": (2.0, 5.0),
+            "subtask_accuracy": (25.0, 35.0),
+        },
+        "hle": {
+            "judge_correct": (4.0, 12.0),
+            "symbolic_correct": (1.0, 10.0),
+        },
+    },
+    "reasoning_off": {
+        "math-500": (70.0, 80.0),
+        "aime24": (11.0, 21.0),
+        "aime25": (3.0, 9.0),
+        "gpqa": (46.0, 56.0),
+        "mmlu-pro": (65.0, 75.0),
+        "livecodebench": (25.0, 35.0),
+        "scicode": {
+            "problem_accuracy": (0.0, 3.0),
+            "subtask_accuracy": (15.0, 25.0),
+        },
+        "hle": {
+            "judge_correct": (2.5, 5.0),
+            "symbolic_correct": (0.5, 2.5),
+        },
+    },
+}
+
+TOOLCALLING_METRIC_PATHS = {
+    "overall_accuracy": ["overall_accuracy", "accuracy"],
+    "overall_non_live": ["non_live_single_turn", "overall_non_live", "accuracy"],
+    "non_live_ast": ["non_live_single_turn", "non_live_ast", "accuracy"],
+    "irrelevance": ["non_live_single_turn", "irrelevance", "accuracy"],
+    "overall_live": ["live_single_turn", "overall_live", "accuracy"],
+    "live_ast": ["live_single_turn", "live_ast", "accuracy"],
+    "live_irrelevance": ["live_single_turn", "live_irrelevance", "accuracy"],
+    "live_relevance": ["live_single_turn", "live_relevance", "accuracy"],
+    "overall_multi_turn": ["multi_turn", "overall_multi_turn", "accuracy"],
+}
+
+TOOLCALLING_METRIC_RANGES = {
+    "reasoning_on": {
```

Comment:
@shtoshni what's the variance here? Current feels too broad, should it be something like 5 points around mean?